falcon/response.py

```diff
@@ -434,3 +434,10 @@ def _wsgi_headers(self, media_type=None):
             return headers.items()
 
         return list(headers.items())  # pragma: no cover
+
+    def redirect(self, uri, permanent=False):
+        if permanent:
+            self.status = "301 Moved Permanently"
```

Comment:
Nitpick: This string could be replaced by `falcon.HTTP_301` or `falcon.HTTP_MOVED_PERMANENTLY`.